scripts/ddd/ddd

```diff
@@ -95,7 +95,11 @@ restore_mapping() {
 
 	# If we have stored USB_MAP, reload it
 	if [ -f ${DDD_MAP} ]; then
-		source ${DDD_MAP}
+        source ${DDD_MAP}
```

Comment:
hmm, I think I see what you're saying it's set out of scope, it kind of sucks that theres no other way to do this :/

Thanks! =) Bash is pretty much my favorite language so I try really hard to make it nice!